src/sentry/static/sentry/app/stores/pluginsStore.tsx

```diff
@@ -1,8 +1,41 @@
 import Reflux from 'reflux';
 
 import PluginActions from 'app/actions/pluginActions';
+import {Plugin} from 'app/types';
+
+type PluginStoreInterface = {
+  state: {
+    loading: boolean;
+    plugins: Plugin[];
+    error: Error | null;
+    pageLinks: string | null;
+  };
+  plugins: Map<string, Plugin> | null;
+  updating: Map<string, Plugin>;
+};
+
+const PluginStoreConfig: Reflux.StoreDefinition & PluginStoreInterface = {
+  plugins: null,
+  state: {
+    loading: true,
+    plugins: [],
+    error: null,
+    pageLinks: null,
```

Comment:
wouldn't be better to have it outside as a const? so then we could reuse it here and inside the reset function :)